lib/squadster/helpers/dates.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Squadster.Helpers.Dates do
+  def date_from_string(string_date) do
+    [day, month, year] = string_date \
```

Comment:
backslash is useless
![image](https://user-images.githubusercontent.com/30197491/73727752-26597800-4743-11ea-8c2a-fd1972c8113c.png)
![image](https://user-images.githubusercontent.com/30197491/73727768-2bb6c280-4743-11ea-998f-38aa632f5611.png)

Reply:
👍